src/key.cpp

```diff
@@ -197,7 +197,7 @@ bool CKey::VerifyPubKey(const CPubKey& pubkey) const {
 bool CKey::SignCompact(const uint256 &hash, std::vector<unsigned char>& vchSig) const {
     if (!fValid)
         return false;
-    vchSig.resize(65);
+    vchSig.resize(CPubKey::COMPRESSED_SIZE);
```

Comment:
This change looks like it'll result in a buffer overflow?

Reply:
Thanks for the catch. The 65 here is the signature size, not the pubkey compressed size. PR updated